docs/source/libraries.rst

```diff
@@ -490,6 +490,36 @@ original signature, thus blinding type checkers and other tools that
 provide signature assistance. As such, library authors are discouraged
 from creating decorators that mutate function signatures in this manner.
 
+Aliasing Decorators
+-------------------
+
+When writing a library with a couple of decorator factories 
+(i.e. functions returning decorators, like ``complex_decorator`` from 
+Annotating Decorators section) it may be tempting to create a shortcut 
```

Comment:
Please add the following line before the "Annotating Decorators" section heading:

```
.. _annotating-decorators:
```

(And a similar line above the "Aliasing Decorators" header.) See the "How much of my library needs types" section for an example.

You can then link the section like this:

```suggestion
:ref:`annotating-decorators` section) it may be tempting to create a shortcut 
```